@xen-orchestra/web-core/lib/components/UiChip.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <span class="ui-chip typo p3-regular" :class="[color, { disabled }]">
```

Comment:
```suggestion
  <span :class="[color, { disabled }]" class="ui-chip typo p3-regular" @click="emit('edit')">
```

I added the `@click`, instead of on the label. Because when moving the mouse over the right part of the tag, it will get the ":hover" and ":active" states, but will not trigger the click event.